Frontend/src/Components/PPC_MPPT/PPC_MPPT.js

```diff
@@ -34,43 +37,135 @@ export default function PPC_MPPT(props) {
                     <Spacer/>
                     <Flex w='45%' direction='column'>
                         <MPPT_CELL
-                            label='Charge'
+                            label='Tracking'
                             isCurrMode={props.data?.mppt_mode[0]}
                         />
                         <Spacer/>
                         <MPPT_CELL
-                            label='MPPT'
+                            label='Current Control'
                             isCurrMode={!props.data?.mppt_mode[0]}
                         />
                     </Flex>
                 </Flex>
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 1 Temp'
-                    data={props.data?.string1_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string1_temp.MIN}
-                    max={CONSTANTS.string1_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string2_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string2_temp.MIN}
-                    max={CONSTANTS.string2_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string3_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string3_temp.MIN}
-                    max={CONSTANTS.string3_temp.MAX}
-                />
+                <Grid
+                    py='1rem'
+                    templateColumns="repeat(4, 1fr)"
+                    templateRows="repeat(4, 1fr)"
+                    gap={0.5}
+                >
+                    <Spacer></Spacer>
+                    <GridItem
+                        bg='gray'
+                        color='black'
+                        fontWeight='bold'
+                        height='20px'
+                        fontSize={10}
+                        alignItems='center'
+                        display='flex'>Temp(ºC)</GridItem>
```

Comment:
```suggestion
                    <GridItem
                        bg='gray'
                        color='black'
                        fontWeight='bold'
                        height='20px'
                        fontSize={10}
                        alignItems='center'
                        display='flex'
                    >
                        Temp(ºC)
                    </GridItem>
```
One more nit about the formatting of components with props that are put on multiple lines. To further improve readability, we generally keep the contents on its own line(s) and then format the opening and closing tags (`<GridItem></GridItem>`) like in the suggestion